src/bn_pending_txns.erl

```diff
@@ -347,7 +348,11 @@ nonce_info(#blockchain_txn_stake_validator_v1_pb{address = Address}) ->
 nonce_info(#blockchain_txn_transfer_validator_stake_v1_pb{old_address = Address}) ->
     {Address, 0, none};
 nonce_info(#blockchain_txn_unstake_validator_v1_pb{address = Address}) ->
-    {Address, 0, none}.
+    {Address, 0, none};
+nonce_info(#blockchain_txn_state_channel_open_v1_pb{owner = Address, nonce = Nonce}) ->
+    {Address, Nonce, oui};
```

Comment:
this is incorrect.. the affected nonce is the balance nonce of the owner, not an "oui nonce"

Reply:
Actually, it looks like API reports a "dc_nonce"? `1b4kPZnsmLQEPaGVL8TVrNDdKaQ4stRYF8bFA4eoi8Bq5aKrAK6` open and closes state channels quite a bit.

```
// 20220128061539
// https://testnet-api.helium.wtf/v1/accounts/1b4kPZnsmLQEPaGVL8TVrNDdKaQ4stRYF8bFA4eoi8Bq5aKrAK6

{
  "data": {
    "staked_balance": 0,
    "speculative_sec_nonce": 0,
    "speculative_nonce": 0,
    "sec_nonce": 0,
    "sec_balance": 0,
    "nonce": 0,
    "dc_nonce": 609,
    "dc_balance": 10193505424,
    "block": 283674,
    "balance": 0,
    "address": "1b4kPZnsmLQEPaGVL8TVrNDdKaQ4stRYF8bFA4eoi8Bq5aKrAK6"
  }
}```

Reply:
right, sorry, it's the dc_nonce